app/javascript/courses/students/components/CoursesStudents__LevelDistribution.re

```diff
@@ -0,0 +1,125 @@
+[%bs.raw {|require("./CoursesStudents__LevelDistribution.css")|}];
+
+open CoursesStudents__Types;
+let str = React.string;
+
+let stylingForLevelPills = percentageStudents => {
+  let emptyStyle = ReactDOMRe.Style.make();
+  let styleWithWidth =
+    ReactDOMRe.Style.make(
+      ~width={
+        (percentageStudents |> Js.Float.toString) ++ "%";
+      },
+      (),
+    );
+  if (0.0 <= percentageStudents && percentageStudents <= 4.0) {
+    ("w-8 flex-shrink-0", emptyStyle, "bg-green-200 text-green-800");
+  } else if (4.0 < percentageStudents && percentageStudents <= 20.0) {
+    ("", styleWithWidth, "bg-green-300 text-green-800");
+  } else if (20.0 < percentageStudents && percentageStudents <= 40.0) {
+    ("", styleWithWidth, "bg-green-400 text-green-900");
+  } else if (40.0 < percentageStudents && percentageStudents <= 60.0) {
+    ("", styleWithWidth, "bg-green-500 text-white");
+  } else if (60.0 < percentageStudents && percentageStudents <= 80.0) {
+    ("", styleWithWidth, "bg-green-600 text-white");
+  } else {
+    ("", styleWithWidth, "bg-green-700 text-white");
+  };
+};
+
+[@react.component]
+let make = (~levels, ~selectLevelCB) => {
+  let totalStudentsInCourse =
+    levels |> Array.fold_left((x, y) => x + Level.studentsInLevel(y), 0);
+  let completedLevels = Level.levelsCompletedByAllStudents(levels);
+  totalStudentsInCourse > 0
+    ? <div
+        ariaLabel="Students level-wise distribution"
+        className="w-full pt-10 max-w-3xl mx-auto hidden md:block">
```

Comment:
`pt-8` instead of `pt-10`, maybe? The gap above the bar feels just a _bit_ too wide.

```suggestion
        className="w-full pt-8 max-w-3xl mx-auto hidden md:block">
```